.gitignore

```diff
@@ -20,6 +20,7 @@ debug.log
 /js/*.db.d
 tool/deployments/*/data
 node_modules
+.idea/
```

Comment:
Can't you just add this to your own global excludes file?

https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_excludesfile

Reply:
thanks you @glouvigny for the tips, it's fixed :)